src/listener.ts

```diff
@@ -359,7 +359,11 @@ class Listener extends EventEmitter implements InterfaceListener {
             return cOpts.host !== host || cOpts.port !== port
           })
 
-    this.externalAddress = await getExternalIp(usableStunServers, this.udpSocket)
+    try {
+      this.externalAddress = await getExternalIp(usableStunServers, this.udpSocket)
+    } catch (err) {
+      error(err.message)
```

Comment:
Why is this better than letting the exception run its course?

Reply:
Currently, it's not properly defined. The idea is to automatically fallback to "not getting a STUN response", hence no external address known, and then log the error message